src/ol/source/wmtssource.js

```diff
@@ -350,8 +351,23 @@ ol.source.WMTS.optionsFromCapabilities = function(wmtsCap, config) {
   /** @type {!Array.<string>} */
   var urls = [];
   var requestEncoding = config['requestEncoding'];
-  if (wmtsCap['OperationsMetadata'].hasOwnProperty('GetTile') &&
-      requestEncoding != 'REST') {
+  if (!requestEncoding) {
```

Comment:
Please use `goog.isDef` if you want to test whether `requestEncoding` is defined.